opentelemetry-sdk/src/metrics/meter.rs

```diff
@@ -107,34 +116,41 @@ impl SdkMeter {
     {
         let validation_result = validate_instrument_config(builder.name.as_ref(), &builder.unit);
         if let Err(err) = validation_result {
-            global::handle_error(err);
+            otel_error!(
+                    name: "SdkMeter.CreateObservableCounter.InstrumentCreationFailed", 
+                    meter_name = self.scope.name.as_ref(),
+                    instrument_name = builder.name.as_ref(),
+                    error = format!("Measurements from the instrument will be ignored. Reason: {}", err));
             return ObservableCounter::new();
         }
 
         match resolver.measures(
             InstrumentKind::ObservableCounter,
-            builder.name,
+            builder.name.clone(),
             builder.description,
             builder.unit,
             None,
         ) {
             Ok(ms) => {
                 if ms.is_empty() {
+                    otel_error!(
+                        name: "SdkMeter.CreateObservableCounter.InstrumentCreationFailed",
+                        meter_name = self.scope.name.as_ref(),
+                        instrument_name = builder.name.as_ref(),
+                        message = "Measurements from the instrument will be ignored. Reason: View Configuration / Drop Aggregation"
+                    );
                     return ObservableCounter::new();
                 }
-
                 let observable = Arc::new(Observable::new(ms));
-
                 for callback in builder.callbacks {
                     let cb_inst = Arc::clone(&observable);
                     self.pipes
                         .register_callback(move || callback(cb_inst.as_ref()));
                 }
-
                 ObservableCounter::new()
             }
-            Err(err) => {
-                global::handle_error(err);
+            Err(_err) => {
+                // TBD - Add error handling
```

Comment:
Let's be consistent with the usage of `TODO`/`TBD` in the repo so that we can search for these actionable items easily. I did a [quick search](https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-rust%20TODO&type=code) on the repo and mostly we have been using `TODO`. There is only one instance where we have used `TBD`.

```suggestion
                // TODO - Add error handling
```

Reply:
Yes, TODO is something we follow in repo, just that I am used to TBD :)